app/src/main/java/com/kuit/kuit6android/ui/home/component/recent.kt

```
@@ -0,0 +1,92 @@
+                    Text(
+                        text = "4.9 (3,849)",
+                        fontSize = 12.sp,
+                        color = Color(0xFF575252)
+                    )
+                }
+            }
+        }
+    }
+}
+
+fun column(modifier: Modifier, function: () -> Unit) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)